WrightTools/_base.py

```diff
@@ -191,6 +209,9 @@ def __version__(self):
             self.file.attrs['__version__'] = wt5_version
         return self.file.attrs['__version__']
 
+    def _update(self):
+        pass
```

Comment:
Does this serve a purpose?

Reply:
just as a placeholder

eventually want to migrate a bunch of natural naming code here